src/main/java/seedu/masslinkers/logic/parser/AddCommandParser.java

```diff
@@ -1,6 +1,7 @@
 package seedu.masslinkers.logic.parser;
```

Comment:
This bug is also affecting `mod add `and `mod delete`
`mod add t23 x`, `mod delete t23 x `gives `Index cannot be empty! Specify the index of a batchmate to be modified.` and not invalid index

addInt are deleteInt do not have this bug.

Reply:
Fixed ✅